_rules/auto-play-audio-has-control-mechanism-4c31df.md

```diff
@@ -251,7 +251,7 @@ The `<video>` has a [mechanism](https://www.w3.org/TR/WCAG21/#dfn-mechanism) to
 The `<video>` element is `muted`.
```

Comment:
This test case appears in other rule i.e. 80f0bf but the description is slightly different here. For example description in inapplicable example 1 of ACT-R 80f0bf is 'The `<video>` element audio autoplays for longer than 3 seconds, but is `muted`.'